examples/pointcloud.js

```diff
@@ -0,0 +1,164 @@
+/* global itowns, pointcloud, GuiTools, view, menuGlobe, debug */
+
+// eslint-disable-next-line no-unused-vars
+function showPointcloud(serverUrl, fileName, lopocsTable) {
+    document.getElementById('viewerDiv').style.display = 'block';
+    document.getElementsByTagName('body').className = '';
+
+    const viewerDiv = document.getElementById('viewerDiv');
+
+    itowns.THREE.Object3D.DefaultUp.set(0, 0, 1);
+
+    itowns.proj4.defs('EPSG:3946',
+        '+proj=lcc +lat_1=45.25 +lat_2=46.75 +lat_0=46 +lon_0=3 +x_0=1700000 +y_0=5200000 +ellps=GRS80 +towgs84=0,0,0,0,0,0,0 +units=m +no_defs');
+
+    window.menuGlobe = new GuiTools('menuDiv');
+
+    window.view = new itowns.View('EPSG:3946', viewerDiv, { renderer: { logarithmicDepthBuffer: false } });
```

Comment:
It caused rendering issues. I'll add a TODO to make sure we revisit this decision.